app/controllers/api/samples_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::SamplesController < ApplicationController
+  before_action :set_user, only: [:show, :update]
+    
+    def index
+        @users = User.all
```

Comment:
if you don't use the variable in a view (be it html or jbuilder) then you don't need to use an instance variable and you can just use a regular variable instead